stable/redis-ha/Chart.yaml

```diff
@@ -5,17 +5,19 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 2.0.1
+version: 3.0.0
 description: Highly available Redis cluster with multiple sentinels and standbys.
 icon: https://upload.wikimedia.org/wikipedia/en/thumb/6/6b/Redis_Logo.svg/1200px-Redis_Logo.svg.png
 maintainers:
-  - email: ikaboubi@gmail.com
-    name: smileisak
-  - email: ianmaddox@google.com
-    name: ianmaddox
+- email: ikaboubi@gmail.com
+  name: smileisak
+- email: ianmaddox@google.com
+  name: ianmaddox
+- email: bzumhagen@gmail.com
+  name: Ben Zumhagen
```

Comment:
@bzumhagen Please also add an [OWNERS](https://github.com/kubernetes/charts#owning-and-maintaining-a-chart) file while you're at it

Reply:
Done